xml/System.Text.Json/JsonWriterOptions.xml

```diff
@@ -55,9 +60,17 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Defines whether the <see cref="T:System.Text.Json.Utf8JsonWriter" /> should skip structural validation and allow
+            the user to write invalid JSON, when set to true. If set to false, any attempts to write invalid JSON will result in
+            a <exception cref="T:System.InvalidOperationException" /> to be thrown.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+            If the JSON being written is known to be correct,
+            then skipping validation (by setting it to true) could improve performance.
+            An example of invalid JSON where the writer will throw (when SkipValidation
+            is set to false) is when you write a value within a JSON object
```

Comment:
There are `true`s and `false`s here (and elsewhere in this change) that are text vs any sort of keyword.  (If there's a style rule that says they should just be text, that's fine; but it doesn't match my expectation :smile:)

Reply:
This section was actually not rendering very well so I switched this to Markdown